.github/workflows/build_conda_pkg.yml

```diff
@@ -1,6 +1,8 @@
 name: Build Conda Package
 
 on:
+  pull_request:
+    types: [ opened, synchronize ]
```

Comment:
Do we want to keep this for feature builds moving forward?

Reply:
We can try it out and if it's too slow we can undo the change?

Reply:
Sure I am a fan!